data-model/SDP-sample-records/writing-system_sample1.json

```diff
@@ -0,0 +1,6 @@
+{
+  "id": "https://sinaimanuscripts.library.ucla.edu/writing-system/1",
+  "label": "Syriac (writing system)",
+  "associated_languages": ["Syriac", "Arabic", "Turkish", "Malayalam"],
+  "contributor": ["gkessel"]
+}
```

Comment:
As you can see, I'm not really sure what to do with writing-system or script. 
We could link to other authority files. E.g., ISO 15924? But these are called 'script codes', and what we call script are 'typographic variants'. Cf. https://en.wikipedia.org/wiki/ISO_15924#Syrc

We could add something like 'dates-in-use', but does that make sense for a writing system?

Employed scripts could be included here? (this is that same question of where we need to specify two-way vs one-way connections, and which direction one-way connections should flow)